### PR TITLE
CONTRIBUTING.md updates

### PR DESCRIPTION
**Pull request by amcamd**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,98 @@
-## How to contribute
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Contributing to hipBLAS">
+  <meta name="keywords" content="ROCm, contributing, hipBLAS">
+</head>
```
Comment (TorreZuk) [inline .github/CONTRIBUTING.md]: HTML to be removed ?

**Review by TorreZuk — APPROVED** [review]
Looks good to me, maybe switch to .rst to match rocBLAS?  Or should rocBLAS change?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,98 @@
...
+    we can collect information about your configuration. This helps reduce the time required to
+    reproduce your issue.
+  * Check your issue regularly, as we may require additional information to successfully reproduce the
+    issue.
+* You may also open an issue to ask questions to the maintainers about whether a proposed change
+  meets the acceptance criteria, or to discuss an idea pertaining to the library.
+
+## Pull Request Guidelines ##
+
+By creating a pull request, you agree to the statements made in the [code license](#code-license) section. Your pull request
```
Comment (daineAMD) [inline .github/CONTRIBUTING.md]: This link doesn't work for me; maybe it works after merge though. Or have to check .md links/.rst links if we switch to rst

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -1,13 +1,98 @@
...
+git ls-files -z *.cc *.cpp *.h *.hpp *.cl *.h.in *.hpp.in *.cpp.in | xargs -0 clang-format -style=file -i
+```
+
+Also, githooks can be installed to format the code per-commit:
+
+./.githooks/install
+
+## Code License ##
+
+All code contributed to this project will be licensed under the license identified in the [LICENSE.md](../LICENSE.md). Your contribution will be accepted under the same license.
```
Comment (daineAMD) [inline .github/CONTRIBUTING.md]: This link also doesn't work for me, looks like rocBLAS uses full path.

**Review by daineAMD — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (amcamd) [reply .github/CONTRIBUTING.md]: The HTML is copied from the template.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (amcamd) [reply .github/CONTRIBUTING.md]: The link works for me, I stayed with .md, did not change to .rst.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (amcamd) [reply .github/CONTRIBUTING.md]: The link works for me.

**Comment by amcamd** [comment]
rocBLAS CONTRIBUTING.rst is included in the documentation file Contributors_Guide.rst. It is best to have .rst in this case rather than .md. For hipBLAS I think it is simpler to stay with .md.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (amcamd) [reply .github/CONTRIBUTING.md]: Switched to .rst, HTML removed.

**Pull request merged**